engine/src/flutter/shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -748,7 +749,7 @@ public AccessibilityNodeInfo createAccessibilityNodeInfo(int virtualViewId) {
       result.addAction(AccessibilityNodeInfo.ACTION_SET_TEXT);
     }
 
-    if (semanticsNode.hasFlag(Flag.IS_BUTTON) || semanticsNode.hasFlag(Flag.IS_LINK)) {
```

Comment:
adding this was a workaround so that people get a hint that this ANI is clickable. Since now the talkback will pronounce the urlspan, this is no longer needed